data/xml/2022.lrec.xml

```diff
@@ -770,7 +770,7 @@
       <pwcdataset url="https://paperswithcode.com/dataset/imdb-movie-reviews">IMDb Movie Reviews</pwcdataset>
     </paper>
     <paper id="63">
-      <title><fixed-case>N</fixed-case>aija<fixed-case>S</fixed-case>enti: A Nigerian <fixed-case>T</fixed-case>witter Sentiment Corpus for Multilingual Sentiment Analysis</title>
+      <title><fixed-case>N</fixed-case>aija<fixed-case>S</fixed-case>enti: A <fixed-case>N</fixed-case>igerian <fixed-case>T</fixed-case>witter Sentiment Corpus for Multilingual Sentiment Analysis</title>
```

Comment:
This should also be added to bin/fixedcase/truelist.

Reply:
First time encounter this. When should we manually add to the bin/fixedcase/truelist?

Reply:
any proper name that is likely to recur